.gitmodules

```diff
@@ -25,3 +25,6 @@
 [submodule "ThirdParty/vtkGUISupportQt"]
 	path = ThirdParty/vtkGUISupportQt
 	url = https://github.com/ufz/vtkGUISupportQt.git
+[submodule "ThirdParty/metis"]
+	path = ThirdParty/metis
+	url = https://github.com/envinf/metis.git
```

Comment:
I'd prefer not to touch the metis code and then execute it via a system call ("externally").

If metis changes we are not affected.